west.yml

```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 22d60f831a43f988fbb9c4475dd4a5ae657930aa
+      revision: pull/581/head
```

Comment:
Please rebase and update SHA: `c1fe5a41c45f41adcfc008fe0ae2a4265e148d81`

Reply:
Rebased and updated SHA.